packages/key-did-resolver/package.json

```diff
@@ -34,7 +38,8 @@
   },
   "dependencies": {
     "@stablelib/ed25519": "^1.0.2",
-    "multiformats": "^9.5.2",
+    "multiformats": "^9.7.1",
```

Comment:
Do you really need a newer `multiformats` library?

Reply:
There is no reason to my knowledge. I changed it back.